src/sections/homepage/search-input/SearchInput.tsx

```diff
@@ -0,0 +1,68 @@
+import { useRef, useState } from 'react'
+import { useNavigate } from 'react-router-dom'
+import { Form } from '@iqss/dataverse-design-system'
+import { CloseButton } from 'react-bootstrap'
+import { Search as SearchIcon } from 'react-bootstrap-icons'
```

Comment:
This may be a lower level priority, but I think our goal is to avoid referring to react-bootstrap in the SPA code, and keep it in the design system code, is that right?
After seeing this I noticed that we are importing 'react-bootstrap-icons' in several places, and one instance where we are importing Dropdown from react-bootstrap.

Reply:
Yeah, I will add the CloseButton to the Design System, it looks better that using our Button with a close icon.
About the Dropdown used in the AddData, I didn't know about that, I will address it better at another time.